packages/shared/package.json

```diff
@@ -2,25 +2,58 @@
   "name": "@clerk/shared",
   "version": "0.24.4",
   "description": "Internal package utils used by the Clerk SDKs",
-  "types": "./dist/types/index.d.ts",
-  "main": "./dist/cjs/index.js",
-  "module": "./dist/esm/index.js",
   "sideEffects": false,
   "files": [
     "dist"
   ],
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
   "exports": {
     ".": {
-      "types": "./dist/types/index.d.ts",
-      "import": "./dist/esm/index.js",
-      "require": "./dist/cjs/index.js"
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    },
+    "./*": {
```

Comment:
The different `types` keys are addressing these two:

- https://publint.dev/rules#types_not_exported
- https://publint.dev/rules#export_types_invalid_format